### PR TITLE
LUCENE-8981: update Kuromoji javadocs, adding experimental tags to Di…

### PR DESCRIPTION
**Pull request by msokolov**
…ctionaryBuilder and JapaneseTokenizer ctor

### CODE REVIEW

**Comment by msokolov** [comment]
@mocobeta does this sound right?

File: lucene/analysis/kuromoji/src/java/org/apache/lucene/analysis/ja/JapaneseTokenizer.java [hunk 1]
```diff
@@ -219,9 +219,9 @@ public JapaneseTokenizer(UserDictionary userDictionary, boolean discardPunctuati
   }
 
   /**
-   * Create a new JapaneseTokenizer, supplying a custom system dictionary and unknown dictionary.
-   * <p>
-   * Uses the default AttributeFactory.
+   * <p>Create a new JapaneseTokenizer, supplying a custom system dictionary and unknown dictionary.
+   * This constructor provides an entry point for users that want to constructcustom language models
```
Comment (mocobeta) [inline lucene/analysis/kuromoji/src/java/org/apache/lucene/analysis/ja/JapaneseTokenizer.java]: Seems a whitespace is missing : constructcustom -> construct custom

**Comment by mocobeta** [comment]
Thanks for giving the elaborated DictionaryBuilder Javadoc!

File: lucene/analysis/kuromoji/src/java/org/apache/lucene/analysis/ja/JapaneseTokenizer.java (hunk 1 above)
Comment (msokolov) [reply lucene/analysis/kuromoji/src/java/org/apache/lucene/analysis/ja/JapaneseTokenizer.java]: Thanks for the review; I'll fix and push.

**Pull request merged**